src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -280,17 +280,19 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ venue }) => {
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const backButtonOptionList = ownedVenues.filter(
-    ({ id, name, template, worldId: venueWorldId }) => {
-      if (venueId === id || venue?.worldId !== venueWorldId) {
-        return null;
-      }
-
-      return {
-        name,
-        template,
-      };
-    }
+  const backButtonOptionList = useMemo(
+    () =>
+      ownedVenues?.filter(({ id, name, template, worldId: venueWorldId }) => {
+        if (venueId === id || venue?.worldId !== venueWorldId) {
+          return null;
+        }
+
+        return {
+          name,
+          template,
+        };
```

Comment:
filter only needs `true` or `false` i.e. truthy or falsy values, so just 
```
.filter ( ({ id, name, template, worldId: venueWorldId }) =>  venueId !== id && venue?.worldId === venueWorldId   )
``` 
should work

Reply:
done 4b0a409